### PR TITLE
DT-2634 Use generic service chart for Helm

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: helm_deploy/hmpps-registers/values.yaml [hunk 1]
```diff
@@ -1,5 +1,47 @@
 ---
 # Values here are the same across all environments
 
+generic-service:
+  nameOverride: hmpps-registers
+  replicaCount: 2
```
Comment (petergphillips) [inline helm_deploy/hmpps-registers/values.yaml:6]: I know the `values-prod` was set to `2` but for all our production services it should be `4`.

Comment (petergphillips) [reply helm_deploy/hmpps-registers/values.yaml:6]: note that you'll need to then override for `dev` to be 2

Comment (karenmillermoj) [reply helm_deploy/hmpps-registers/values.yaml:6]: ah ok - I didn't realise that.

File: helm_deploy/hmpps-registers/values.yaml [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+    httpGet:
+      path: /ping
+
+  readinessProbe:
+    httpGet:
+      path: /ping
+
+  env:
+    TOKEN_VERIFICATION_ENABLED: true
+    NODE_ENV: production
```
[the remark this answers is not in the document]
Comment (karenmillermoj) [reply helm_deploy/hmpps-registers/values.yaml:27]: it is funny as I was thinking that - the one in dev shows it without the quotes but the new one adds them in.  I think, for completeness, seeing as it is adding in the quotes, it should match - so yes, will do that too.  Cheers

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**